pkg/utils/slices.go

```diff
@@ -18,10 +18,29 @@
 package utils
 
 import (
+	"errors"
 	"sort"
 	"strings"
 )
 
+// ToMap converts a list of strings to a map
+func ToMap(slice []string) (map[string]string, error) {
+	m := make(map[string]string)
+
+	for _, item := range slice {
+		if item == "" {
+			return nil, errors.New("empty string found in slice")
+		}
+		items := strings.Split(item, "=")
+		if len(items) != 2 {
+			return nil, errors.New("invalid key=value pair found in slice")
+		}
```

Comment:
where would these errors come out, would they be surfaced to a user? Could it be better to log and silently ignore invalid values?

Reply:
eekk .. too late :-D ..